infra/testing/test-server.js

```diff
@@ -1,19 +1,51 @@
 const express = require('express');
 const oneLine = require('common-tags').oneLine;
+const path = require('path');
+const serveIndex = require('serve-index');
 
 const logHelper = require('../utils/log-helper');
 
 const app = express();
 
-app.get('/', (req, res) => {
-  res.send('Hello World!');
+app.get(/\/__WORKBOX\/buildFile\/(workbox-[A-z]*)(\.(?:dev|prod)\.(.*))*/, (req, res) => {
+  const moduleName = req.params[0];
+  const fileExtension = req.params[2];
+
+  const modulePath = path.join(__dirname, '..', '..', 'packages', moduleName);
+  const pkg = require(
```

Comment:
As far as I can tell, `require()` always uses paths separated with `/`, even under Windows. (Correct me if I'm wrong.)

So you can either just hardcode the `/` or use `path.posix.join()`.